akka-testkit-typed/src/main/scala/akka/typed/testkit/TestInbox.scala

```diff
@@ -27,11 +30,22 @@ class Inbox[T](name: String) {
     new FunctionRef[T](path, (msg, self) ⇒ q.add(msg), (self) ⇒ ())
   }
 
+  /**
+   * Get and remove the oldest message
+   */
   def receiveMsg(): T = q.poll() match {
     case null ⇒ throw new NoSuchElementException(s"polling on an empty inbox: $name")
     case x    ⇒ x
   }
 
+  /**
+   * Assert and remove the the oldest message.
+   */
+  def expectMsg(expectedMessage: String): Unit = q.poll() match {
```

Comment:
should we return `this` to allow for fluent api?

Reply:
Yes that makes sense